### PR TITLE
Move QuietBackgroundProcesses UI to Windows customization

### PR DESCRIPTION
**Pull request by nieubank**
## Summary of the pull request
Move QuietBackgroundProcesses UI to Windows customization instead of a top-level page.

## References and relevant issues
image

## Detailed description of the pull request / Additional comments
Merge the UI project for QuietBackgroundProcesses into Customization and control the view's visibility based on the experimental features enablement status.

## Validation steps performed
With the feature disabled in Settings -> Experimental features navigate to Windows customization and see that the Quiet mode card is not visible.

Enable the feature and notice that the settings card is visible upon navigating back to Windows customization.

Start a session, navigate away from the page and back and ensure the session remains active.

Close Dev Home and verify that DevHome.QuietBackgroundProcesses.ElevatedServer.exe is still running, launch, navigate back to Windows customization and stop session. Confirm that everything cleans up.

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

**Comment by jsidewhite** [comment]
I'm good with these changes as-is, but @EricJohnson327 mentioned [here ](https://github.com/microsoft/devhome/pull/2202#discussion_r1480637297) that,

"The QuiteBackgroundProcesses stuff should go in its own tools/QuietBackgroundProcesses project. All code should be written as if it wasn't an experimental feature, then the experimental feature checks just wrap it to enable/disable it. That way when it's not experimental anymore, we just remove the checks and the rest of the code stays exactly the same."

I'm signing off, but giving Eric a chance to respond (I'm not sure if what he's saying is possible for WindCust anyway).

Thanks @nieubank!

**Review by jsidewhite — APPROVED** [review]
:shipit:

File: tools/Customization/DevHome.Customization/ViewModels/QuietBackgroundProcessesViewModel.cs [hunk 1]
```diff
@@ -5,19 +5,22 @@
...
 
 public partial class QuietBackgroundProcessesViewModel : ObservableObject
 {
     private readonly ILogger _log = Log.ForContext("SourceContext", nameof(QuietBackgroundProcessesViewModel));
 
-    private readonly TimeSpan _zero = new TimeSpan(0, 0, 0);
-    private readonly TimeSpan _oneSecond = new TimeSpan(0, 0, 1);
+    private readonly IExperimentationService _experimentationService;
+
+    private readonly TimeSpan _zero = new(0, 0, 0);
```
Comment (AmelBawa-msft) [inline tools/Customization/DevHome.Customization/ViewModels/QuietBackgroundProcessesViewModel.cs:20]: super nit: `TimeSpan.FromSeconds(1)`

File: tools/Customization/DevHome.Customization/Views/QuietBackgroundProcessesView.xaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                    Content="{x:Bind ViewModel.QuietButtonText, Mode=OneWay}"
+                    IsEnabled="{x:Bind ViewModel.IsFeaturePresent, Mode=OneWay}"
+                    IsChecked="{x:Bind ViewModel.QuietButtonChecked, Mode=OneWay}"
+                    Command="{x:Bind ViewModel.QuietButtonClickedCommand}" />
+            </StackPanel>
+
+            <!-- Expander -->
+            <ctControls:SettingsExpander.Items>
+                <ctControls:SettingsCard x:Uid="QuietBackgroundProcessesExplanation" HorizontalContentAlignment="Left" >
+                </ctControls:SettingsCard>
```
Comment (AmelBawa-msft) [inline tools/Customization/DevHome.Customization/Views/QuietBackgroundProcessesView.xaml:38]:
```suggestion
                <ctControls:SettingsCard x:Uid="QuietBackgroundProcessesExplanation" HorizontalContentAlignment="Left" />
```

**Review by AmelBawa-msft — APPROVED** [review]

**Pull request merged**